cfme/networks/cloud_network.py

```diff
@@ -24,7 +24,7 @@
 
 
 @attr.s
-class CloudNetwork(Taggable, BaseEntity):
+class CloudNetwork(Taggable, BaseEntity, CustomButtonEventsMixin):
     """Class representing cloud networks in cfme database"""
     in_version = ('5.8', version.LATEST)
```

Comment:
What is variable doing? I don't see it used in anywhere else in this file.

Reply:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/__init__.py#L711

provides `get_button_events` method for this Entity.

Reply:
Just for visibility, I was referring to the `in_version` variable - I am going to see if this variable is used elsewhere and whether or not it can be removed. Do you happen to know @digitronik?